assignments/arrays.js

```diff
@@ -63,32 +63,51 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('Car 33 is a ' + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model + " " + inventory[32].car_model);
```

Comment:
Daaaaaaaave don't do this! Use template literals they make your life sooooooooooooooo much easier!

Here's an example:
`` console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`); ``

Couple things you'll notice
1) Instead of you using `'` you use `` ` `` when you use `` ` `` it signifies that you going to be using a template literal
2) When using a template literally there is no need for concatenating your string's and variable's you simple just write out plain text and use this `${variableName}` syntax when referencing a variable.

Reply:
You got it on template literals! I knew there was another way, but couldn't remember what it was. I will definitely be using them from now on!